sdk/metric/instrument.go

```diff
@@ -183,6 +184,7 @@ type int64Inst struct {
 	embedded.Int64UpDownCounter
 	embedded.Int64Histogram
 	embedded.Int64Gauge
+	x.EnabledInstrument
```

Comment:
I think we rather want to check if `int64Inst` implements `x.EnabledInstrument` rather than embedding it.

Reply:
Updated, PTAL